sources/assets/exegol/load_supported_setups.sh

```diff
@@ -131,6 +131,33 @@ function deploy_firefox_addons() {
   fi
 }
 
+function deploy_bloodhound_customqueries_replacement() {
+  # Merge all the *.json files in customqueries format from my-resources/.../replacement to replace the Exegol provided one
+  if \
+    [[ -d "$MY_Setup_PATH/bloodhound_customqueries/replacement" ]] && \
```

Comment:
You must handle what happen if not: create directory, copy skel file etc.

Reply:
I have not implemented your request as I need more information about the use case to understand.
I was under the assumption that this code would execute once `when the container is being created`.

Is your comment about covering the case when the container is already living, that the user had manually removed the replacement/merge folders and then runs the script manually ?

Reply:
Ok, I think I now understand a bit better the use case. This request should be implemented the way you want.